eslint.config.mjs

```diff
@@ -3,7 +3,7 @@ import ember from "eslint-plugin-ember";
 
 export default [
   {
-    ignores: ["/blueprints/**", "node_modules/**"],
+    ignores: ["blueprints/**", "node_modules/**"],
```

Comment:
Where did you find this config property? I can not find it at https://eslint.org/docs/v8.x/use/configure/ignore

Would rather restore the `.eslintignore` file.